sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/DefaultConfigProperties.java

```diff
@@ -156,6 +162,29 @@ public Duration getDuration(String name) {
     return filterBlanksAndNulls(value.split(","));
   }
 
+  /**
+   * Returns {@link ConfigProperties#getList(String)} as a {@link Set} after validating there are no
+   * duplicate entries.
+   *
+   * @throws ConfigurationException if {@code name} contains duplicate entries
```

Comment:
is it really a problem if there's a duplicate?

Reply:
I see this was existing behavior from the span exporter config... I wonder why it matters?

Reply:
I can't think of a valid configuration where you would want to use the same exporter multiple times, especially since there's no way to have different config for each of the exporters. Until a use case for that comes up, I think its better to fail to help users avoid an accidental invalid config.

Reply:
It doesn't seem harmful to me if we were to ignore duplicates, but I'm also ok with keeping the existing behavior.